src/base/settingsstorage.cpp

```diff
@@ -65,12 +65,12 @@ namespace
     QString mapKey(const QString &key)
     {
         static const MappingTable keyMapping = {
```

Comment:
>Then I would imagine you read from both old & new key and write with new key.

I'm going to do so after a few releases (ideally when we completely got rid of Preferences). I don't want to change the configuration file now when qBittorrent is still unstable and users rolled back to older releases often.